guide/using-the-pioneer-3-dx-robot.md

```diff
@@ -0,0 +1,52 @@
+## Using the Pioneer 3-DX robot
+
+%figure "Pioneer 3-DX, an all-purpose base, used for research and applications"
+
+![pioneer3dx.png](images/pioneer3dx.png)
+
+%end
+
+The Pioneer 3-DX robot is an all-purpose base, used for research and applications involving mapping, teleoperation, localization, monitoring, reconnaissance and other behaviors.
+Pioneer 3-DX is caracterized by a set of features listed in [this table](#pioneer-3-dx-characteristics).
+
+The base Pioneer 3-DX platform is assembled with motors featuring 500-tick encoders, 19 cm wheels, tough aluminum body, 8 forward-facing ultrasonic (sonar) sensors, 8 optional rear-facing sonars, 1, 2 or 3 hot-swappable batteries, and a complete software development kit.
+The base Pioneer 3-DX platform can reach speeds of 1.6 meters per second and carry a payload of up to 23 kg.
+
+More information on the specifications and optional devices is available on Adept Mobile Robots official [webpage](http://www.mobilerobots.com/ResearchRobots/PioneerP3DX.aspx).
+
+### Pioneer 3-DX model
+
+%figure "Pioneer 3-DX characteristics"
+
+| Characteristics     | Description                                                                     |
+| ------------------- | ------------------------------------------------------------------------------- |
+| Dimensions          | 485 mm long, 381 mm large, 217 mm high                                          |
+| Weight              | 10 kg, operating playload of 17 kg                                              |
```

Comment:
Is it 17 or 23 kg as said previously. That's not clear...

Reply:
The 23 is the max payload according to the constructor. 17kg is probably the one for Webots although i will admit I do not understand fully this "operating payload".

Reply:
I have reported the real mass as it is simulated in Webots (this explains the differences with the previous tables which were referencing the constructor's specifications). The idea behind this change is since the branch is on master the tables should reflect the current state of the simulation. Furthermore, the correct values (according to the previous tables) have been requested in #6765

Reply:
If nobody understands where do these 17 kg come from, I would simply remove them.

Reply:
Maybe @fabienrohrer knows?

Reply:
I didn't do this model. As Olivier said we can simply drop the problematic info from the doc, and add a note in the issue to fix this for long term for Webots R2018a.rev1 (I will soon start to do this kind of modifs)